site/scripts/get-contributors.js

```diff
@@ -45,7 +45,7 @@ async function main() {
 	// TODO: Optimizing the static/contributors.jpg image should probably get automated as well
 	console.log('remember to additionally optimize the resulting /static/contributors.jpg image file via e.g. https://squoosh.app ');
 
-	const str = `[\n\t${authors.map(a => `'${a.login}'`).join(',\n\t')}\n]`;
+	const str = `[\n\t${authors.map(a => `'${a.login.replace('[bot]', '')}'`).join(',\n\t')}\n]`;
```

Comment:
how about we `.filter` out any author with `[bot]` in the name instead of just removing the `[bot]` portion?